app/views/admin/loans/content/_breakeven_charts.html.slim

```diff
@@ -0,0 +1,24 @@
+.summary-chart.breakeven-chart.row
+  h3.col-sm-12 = t("loan.breakeven.charts_title")
+  .chart.col-lg-6.col-sm-12
+    h4 = t("loan.breakeven.revenue_by_product")
+    #breakeven-revenue-chart
+  .chart.col-lg-6.col-sm-12
+    h4 = t("loan.breakeven.production_cost_by_product")
+    #breakeven-production-cost-chart
+  .chart.col-lg-6.col-sm-12
+    h4 = t("loan.breakeven.profit_by_product")
+    #breakeven-product-profit
+  .chart.col-lg-6.col-sm-12
+    h4 = t("loan.breakeven.fixed_costs", count: 2)
+    #breakeven-fixed-cost-chart
+  .chart.col-lg-6.col-sm-12
+    h4 = t('loan.breakeven.total_cost_breakdown')
+    #breakeven-total-costs-chart
+
+<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```

Comment:
No problem. I looked into it a bit and I think it should be inserted into the head by this partial, so it's only called when needed. That might require a bit of rearranging of `yield`/`content_for`s. Also I think it's a little better to use Rails' `javascript_include_tag`.